app/controllers/posts.js

```diff
@@ -74,10 +74,10 @@ export default Controller.extend({
     postListFocused: equal('keyboardFocus', 'postList'),
     postContentFocused: equal('keyboardFocus', 'postContent'),
 
-    sortedPosts: computed('model.@each.status', 'model.@each.publishedAt', 'model.@each.isNew', 'model.@each.updatedAt', function () {
```

Comment:
Ahhh... sorry... Didn't but it back.. I did some changes but reverted them before pushing. Those, I forgot...